docs/_docs/upgrading.md

```diff
@@ -8,3 +8,6 @@ Upgrading from an older version of Jekyll? Upgrading to a new major version of J
 
 - [From 0.x to 1.x and 2.x](/docs/upgrading/0-to-2/)
 - [From 2.x to 3.x](/docs/upgrading/2-to-3/)
+
+If you are making a minor update (for example from 3.3.1 to the latest version at the time 3.3.2) run 'bundle update jekyll' when in your site directory. 
```

Comment:
I don't think this only applies when making a minor update. Depending on the `Gemfile`, this could also even update to a new _major_ version of Jekyll.